test/check-paths.sh

```diff
@@ -30,21 +30,21 @@ exactly 2 "/datum text paths" '"/datum'
 exactly 2 "/mob text paths" '"/mob'
 exactly 10 "/obj text paths" '"/obj'
 exactly 8 "/turf text paths" '"/turf'
-exactly 139 "to_world uses" '\sto_world\('
+exactly 153 "to_world uses" '\sto_world\('
 exactly 0 "globals with leading /" '^/var' -P
 exactly 0 "globals without global sugar" '^var/(?!global/)' -P
 exactly 0 "apparent paths with trailing /" '\w/[,\)\n]' -P
-exactly 56 "to_world_log uses" '\sto_world_log\('
-exactly 0 "world<< uses" 'world<<|world[[:space:]]<<'
-exactly 0 "world.log<< uses" 'world.log<<|world.log[[:space:]]<<'
-exactly 2 "<< uses" '(?<!<)<<(?!<)' -P
-exactly 2 ">> uses" '(?<!>)>>(?!>)' -P
+exactly 82 "to_world_log uses" '\sto_world_log\('
+exactly 1 "world<< uses" 'world<<|world[[:space:]]<<'
+exactly 1 "world.log<< uses" 'world.log<<|world.log[[:space:]]<<'
+exactly 14 "<< uses" '(?<!<)<<(?!<)' -P
+exactly 3 ">> uses" '(?<!>)>>(?!>)' -P
 exactly 0 "incorrect indentations" '^( {4,})' -P
-exactly 26 "text2path uses" 'text2path'
+exactly 37 "text2path uses" 'text2path'
```

Comment:
Indispensable, because it's used by the SerDe to lookup procs from the references embedded in the file data, and there's no real alternative to text2path available (at least as of BYOND 514).